### ISSUE TITLE
Generate Atoms of different types

### ISSUE THREAD

**Issue by AdamRamberg** [AdamRamberg/unity-atoms#60]
**Is your feature request related to a problem? Please describe.**
Make it possible to generate Atoms of different type for: 
- Actions
- Listeners
- Functions
- Events

Eg. generate a BoolIntEvent. This should be done in the Generator already available in Unity Atoms. 

**Describe the solution you'd like**
This is not part of the regular Atoms generation currently implemented in Unity Atoms. I propose to create a new "page" in the Generator to generate "Multi Atoms" (I'm open for better naming here). The user will first pick what Atom to generate (selection box): Action, Listener, Function or Event. Then options to add types will appear. 

**Describe alternatives you've considered**
Another consideration is to create a new Generator for this kind of generation. 

I would be really appreciate someone making a PR on this. Make sure to read the [guidelines](https://github.com/AdamRamberg/unity-atoms/blob/canary/CONTRIBUTING.md) before starting + assign yourself (or ask me) to the issue. Also don't hesitate to ask any further questions below before developing.

**Comment by soraphis** [comment]
superseeded by #495
